src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+body,
+p,
+h1 {
   margin: 0;
 }
+
+ul {
+  margin: 0;
+  margin-block: 0;
+  margin-inline: 0;
```

Comment:
The `margin-inline: 0;` is redundant since `margin: 0;` already sets all margins to zero. Consider removing it to simplify the code.